PocketCastsTests/UnitTests.xctestplan

```diff
@@ -13,6 +13,7 @@
     }
   ],
   "defaultOptions" : {
+    "codeCoverage" : false,
```

Comment:
I had to add this to make SwiftUI previews work. 🤷 

More details on the commit: https://github.com/Automattic/pocket-casts-ios/pull/377/commits/3142e082d89b477bbecfbdaabab60cc11f47c56d